src/main/java/com/github/invictum/reportportal/ReportIntegrationConfig.java

```diff
@@ -80,6 +82,12 @@ public String communicationDirectory() {
         return System.getProperty(COMMUNICATION_DIR_KEY);
     }
 
+    public int retriesCount() {
+        String value = System.getProperty(FAILSAFE_RERUN_KEY);
+        value = value == null ? System.getProperty(SUREFIRE_RERUN_KEY) : value;
+        return value == null ? 0 : Integer.parseInt(value);
+    }
+
```

Comment:
Check my comment regarding changing approach from system properties to config instance.
But if you still sure system properties is the only way, then current method can be simplified.
```
String value = System.getProperty(FAILSAFE_RERUN_KEY, SUREFIRE_RERUN_KEY);
return value == null ? 0 : Integer.parseInt(value);
```

Reply:
Unfortunately i should use system properties. But i change code to `System.getProperty(FAILSAFE_RERUN_KEY, System.getProperty(SUREFIRE_RERUN_KEY));`